.github/dependabot.yml

```diff
@@ -7,18 +7,24 @@ updates:
     directory: "/tdrs-frontend"
     schedule:
       interval: "daily"
-    target-branch: "raft-tdp-main"
+    target-branch: "dependabot-mass-merge"
+    open-pull-requests-limit: 50
     labels:
       - "dependencies"
       - "frontend"
       - "raft review"
+    reviewers:
+      - "raft-tech/tdrs-developers"
 
   - package-ecosystem: "pip"
     directory: "/tdrs-backend"
     schedule:
       interval: "daily"
-    target-branch: "raft-tdp-main"
+    target-branch: "dependabot-mass-merge"
+    open-pull-requests-limit: 50
```

Comment:
Why have a limit?

Reply:
Not setting the limit defaults it to 5 and setting a 0 as the limit turns off updates for that package ecosystem. 50 was just an arbitrary higher number used to let it open more PRs.